web/src/app/profile/approved-researcher/components/AgreementForm.css

```
@@ -1,32 +1,23 @@
-.form-container {
+.form__wrapper {
   max-width: 400px;
   margin: 2rem auto;
   padding: 1rem;
 }
 
-.input-checkbox {
+input[name="agreed"][type="checkbox"] {
   margin: 0.5rem;
 }
 
-.submit-button {
+.button--submit {
   width: 100%;
   padding: 0.5rem;
   margin-top: 1rem;
-  background-color: #2563eb;
   color: white;
   border: none;
   border-radius: 4px;
   cursor: pointer;
-}
-
-.submit-button:hover {
-  background-color: #1d4ed8;
-}
 
-.submitted-box {
-  margin-top: 1rem;
-  padding: 0.5rem;
-  background-color: #171717;
-  border-radius: 4px;
-  font-size: 0.9rem;
+  &:hover {
+    background-color: color-mix(in srgb, var(--cta-green), 60%, white 40%);
+  }
```

```suggestion
  &:hover {
    background-color: color-mix(in srgb, var(--cta-green) 60%, white 40%);
  }
```

I think that is the official syntax?